examples/molecular_dynamics/example_molecular_dynamics.cpp

```diff
@@ -148,7 +150,8 @@ int main(int argc, char *argv[])
               KokkosExt::ArithmeticTraits::infinity<decltype(rsq)>::value;
           if (rsq < cutoff_sq_ij)
           {
-            auto const r2inv = static_cast<decltype(rsq)>(1) / rsq;
+            auto const r2inv =
+                static_cast<std::remove_const_t<decltype(rsq)>>(1) / rsq;
```

Comment:
Why was casting `1` to `const float` problematic?

Reply:
I kept getting a warning that the const qualifer was being ignored.